translations/en/translation.json

```diff
@@ -22,17 +22,24 @@
       "syntax": {
-        "invalidToken": "There's a syntax error due to a symbol that JavaScript doesn't recognize or didn't expect at its place.\nFor more: {{url}}",
-        "unexpectedToken": "There's a syntax error due to a symbol that wasn't expected at its place.\nUsually this is due to a typo. Check the line number in the error below for anything missing/extra.\nFor more: {{url}}"
+        "badReturnOrYield": "\nSyntax Error ▶️ `return` / `yield` lies outside of a function. Make sure you’re not missing any brackets, so that `return` / `yield` lies inside a function.\nFor more: {{url}}",
+        "invalidToken": "There's a syntax error due to a symbol that JavaScript doesn't recognize or didn't expect at it's place.\nFor more: {{url}}",
+        "missingInitializer": "\nSyntax Error ▶️ A const variable is declared but not initialized. In JavaScript, an initializer for a const is required. A value must be specified in the same statement in which the variable is declared. Check the line number in the error below and assign the const variable a value.\nFor more: {{url}}",
+        "redeclaredVariable": "\nSyntax Error ▶️ \"{{symbol}}\" is being redefined. JavaScript doesn't allow declaring a variable more than once. Check the line number in error below for redeclaration of the variable.\nFor more: {{url}}",
+        "unexpectedToken": "There's a syntax error due to a symbol that wasn't expected at it's place.\nUsually this is due to a typo. Check the line number in the error below for anything missing/extra.\nFor more: {{url}}"
       },
       "type": {
+        "constAssign": "\nError ▶️ A const variable is being re-assigned. In javascript, re-assigning a value to a constant is not allowed. If you want to re-assign new values to a variable, make sure it is not declared a const.\n{{location}}\nFor more: {{url}}",
         "notfunc": "There's an error as \"{{symbol}}\" could not be called as a function {{location}}.\nCheck the spelling, letter-casing (JavaScript is case-sensitive) and its type.\nFor more: {{url}}",
-        "notfuncObj": "There's an error as \"{{symbol}}\" could not be called as a function {{location}}.\nVerify whether \"{{obj}}\" has \"{{symbol}}\" in it and check the spelling, letter-casing (JavaScript is case-sensitive) and its type.\nFor more: {{url}}"
+        "notfuncObj": "There's an error as \"{{symbol}}\" could not be called as a function {{location}}.\nVerify whether \"{{obj}}\" has \"{{symbol}}\" in it and check the spelling, letter-casing (JavaScript is case-sensitive) and its type.\nFor more: {{url}}",
+        "readFromNull": "\nError ▶️ The property of null can't be read. In javascript the value null indicates that an object has no value.\nFor more: \n{{url1}}\n{{url2}}",
+        "readFromUndefined": "\nError ▶️ The property of something undefined can't be read. Check the line number in error below and make sure the variable which is being operated is not undefined.\nFor more: \n{{url1}}\n{{url2}}"
```

Comment:
I wonder if this message could be `"\nError ▶️ \"{{symbol}}\" is undefined, the property of undefined cannot be read`

Reply:
@lm-n we don't have access to {{symbol}} in this error because the browser itself doesn't provide it. 
What if we change it to:
`\nError ▶️ cannot read property of undefined. Check the line number in the error and make sure the variable which is being operated is not undefined.\nFor more: \n{{url1}}\n{{url2}}`

Reply:
sounds good to me!